xml/net/client/protocol.xml

```diff
@@ -162,7 +162,7 @@
     <packet family="Character" action="Remove">
         <comment>Confirm deleting character from an account</comment>
         <field name="session_id" type="short"/>
-        <field name="character_id" type="int"/>
+        <field name="character_id" type="short"/>
```

Comment:
Further investigation has revealed that the client sends a variable sized field for this.
- < short_max: 2 bytes
- < three_max: 3 bytes
- otherwise: 4 bytes

Do we have a way to model this in the protocol?

Reply:
Added `varint` type and accompanying documentation to represent this.